packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -1866,6 +1868,13 @@ Profile for any Group that is not imported from Active Directory
   "description": "All Users West of The Rockies"
 }
 ```
+##### Custom Profile Properties
+
+<ApiLifecycle access="ea" />
+
+Group profiles may be extended with custom properties but the property must first be added to the group profile schema before it can be referenced.  You can use the Profile Editor in the administrator UI or the [Schemas API](/docs/reference/api/schemas/) to manage schema extensions.
```

Comment:
```suggestion
Group Profiles may be extended with custom properties but the property must first be added to the Group Profile schema before it can be referenced.  You can use the Profile Editor in the administrator UI or the [Schemas API](/docs/reference/api/schemas/) to manage schema extensions.
```